kernel/src/process_printer.rs

```diff
@@ -17,7 +14,7 @@ pub struct ProcessPrinterContext {
     /// The overall print message is broken in to chunks so that it can be fit
     /// in a small buffer that is called multiple times. This tracks which byte
     /// we are at so we can ignore the text before and print the next bytes.
-    offset: usize,
+    pub offset: usize,
```

Comment:
Same visibility comment here; more motivated by this being a somewhat tightly coupled interface than any concerns regarding access.